ocaml/idl/datamodel_repository.ml

```diff
@@ -94,6 +95,28 @@ let introduce_bundle =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_remote_pool =
+  call ~name:"introduce_remote_pool" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Add the configuration for a new remote pool repository"
```

Comment:
How about `Add a repository which is synced and applied by a remote pool`?

Reply:
Yes, we can sync updates only when the remote pool is synced and applied, but I think the configuration is only about the configuration of the remote pool, and the state of the pool can change over time.

Reply:
The purpose here is to let users understand what's a `remote pool repository`.